ops/services/frontend/main.tf

```diff
@@ -12,7 +12,7 @@ resource "azurerm_storage_account" "frontend" {
   name                      = "simplereport${var.env}frontend"
   resource_group_name       = var.rg_name
   enable_https_traffic_only = true
-  min_tls_version           = TLS1_2
+  min_tls_version           = "TLS1_2"
```

Comment:
⚠️ noticed [all the other versions were strings](https://github.com/search?q=repo%3ACDCgov%2Fprime-simplereport%20TLS1_2&type=code) and not a variable which [caused this failure initially](https://github.com/CDCgov/prime-simplereport/actions/runs/11427534880/job/31791773879)